src/main/webapp/scripts/user-directory.js

```diff
@@ -20,11 +20,12 @@ function generateUserCards() {
       userCardElement.className = "user-card";
       const profilePictureHTML = "<img class=\"user-img\" src=\"" + user.profilePictureUrl + "\"></img>";
```

Comment:
nitty nit: can you please rename the css class to `profile-picture` to match HTML element name and also be consistent with the one in `my-profile`